main.go

```diff
@@ -67,6 +67,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&opt.Destination, "destination", "stdout", "Log Destination: loki, elasticsearch, stdout, file. (default stdout)")
 	rootCmd.PersistentFlags().Int64Var(&opt.TotalLogLines, "totalLogLines", 0, "Total number of log lines per thread (default 0 - infinite)")
 	rootCmd.PersistentFlags().BoolVar(&opt.DisableSecurityCheck, "disable-security-check", false, "Disable security check in HTTPS client. (default false)")
+	rootCmd.PersistentFlags().StringVar(&opt.BearerTokenFile, "bearer-token-file", "/var/run/secrets/kubernetes.io/serviceaccount/token", "bearer token file (default service account token)")
```

Comment:
Just looked at how it looks when compiled/run and the default is automatically added by the flag library (this is true for the other flags as well, but I wouldn't do the cleanup in the same PR :slightly_smiling_face: ):

```suggestion
	rootCmd.PersistentFlags().StringVar(&opt.BearerTokenFile, "loki-bearer-token-file", "/var/run/secrets/kubernetes.io/serviceaccount/token", "Path to file containing bearer token")
```

I have also prefixed the flag name with `loki-` because it is only used with that output mode. Maybe we can also shorten it to `loki-token-file`, if the name is too long now.